dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05200WorkflowTaskUniqueKeyTest.java

```diff
@@ -49,4 +65,69 @@ public void Test_Upgrade_Task() throws DotDataException {
         task.executeUpgrade();
     }
 
+    @Test
+    public void Test_Upgrade_Task_When_Duped_WorkflowTasks_Exists() throws DotDataException {
+
+        removeConstraintIfAny();
+
+        final User systemUser = APILocator.systemUser();
+        final Language language = APILocator.getLanguageAPI().getDefaultLanguage();
+        final WorkflowAPI workflowAPI = APILocator.getWorkflowAPI();
+
+        final Contentlet contentlet = TestDataUtils.getPageContent(true, language.getId());
+
+        //save workflow task
+        final WorkflowStep workflowStep           = workflowAPI.findStep(
+                SystemWorkflowConstants.WORKFLOW_NEW_STEP_ID);
+        workflowAPI.deleteWorkflowTaskByContentletIdAnyLanguage(contentlet, systemUser);
+        final WorkflowTask workflowTask = workflowAPI
+                .createWorkflowTask(contentlet, systemUser, workflowStep, "test", "test");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "test" appears 4 times in this file; the first occurrence is on line 84](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4666366)